evm_arithmetization/src/generation/state.rs

```diff
@@ -254,6 +255,21 @@ pub(crate) trait State<F: Field> {
         let opcode = read_code_memory(generation_state, &mut row);
         (row, opcode)
     }
+
+    /// Logs `msg` in `debug` mode, in the interpreter.
```

Comment:
The comments here and below are a bit misleading, as this is a default implem for the trait, so not specific to the interpreter.